frontend/src/common/components/mentoringForm/ProfileSection/ProfileSection.tsx

```diff
@@ -0,0 +1,166 @@
+import styled from '@emotion/styled';
+
+import deleteIcon from '../../../../common/assets/images/deleteIcon.svg';
+import uploadIcon from '../../../../common/assets/images/uploadIcon.svg';
+import usePreviewImage from '../../../hooks/usePreviewImage';
+import TitleSeparator from '../TitleSeparator/TitleSeparator';
+
+interface ProfileSectionProps {
+  onProfileImageChange: (file: File | null) => void;
+  profileImageUrl?: string | null;
+}
+
+function ProfileSection({
+  profileImageUrl,
+  onProfileImageChange,
+}: ProfileSectionProps) {
+  const { previewUrl, handleImageChange, updatePreviewUrl } =
+    usePreviewImage(profileImageUrl);
+
+  const handleProfileImageInputChange = (
+    e: React.ChangeEvent<HTMLInputElement>,
+  ) => {
+    handleImageChange(e);
+    onProfileImageChange(e.target.files?.[0] || null);
+  };
+
+  const handleDeleteProfileImageClick = () => {
+    updatePreviewUrl('');
+    onProfileImageChange(null);
+  };
+
+  return (
+    <section>
+      <TitleSeparator>프로필 사진</TitleSeparator>
+      <StyledProfileWrapper>
+        <StyledDeleteButton
+          type="button"
+          onClick={handleDeleteProfileImageClick}
+        >
+          <img src={deleteIcon} alt="삭제 아이콘" />
+        </StyledDeleteButton>
+        <StyledProfileInputWrapper>
+          {previewUrl ? (
+            <>
+              <StyledHiddenInput
+                type="file"
+                accept="image/*"
+                id="profileImage"
+                onChange={handleProfileImageInputChange}
+              />
+              <StyledPreviewImage src={previewUrl} alt="프로필 사진 미리보기" />
+            </>
+          ) : (
+            <>
+              <StyledHiddenInput
+                type="file"
+                accept="image/*"
+                id="profileImage"
+                onChange={handleProfileImageInputChange}
+              />
+
+              <StyledContentWrapper>
+                <StyledUploadIcon src={uploadIcon} alt="업로드 아이콘" />
+                {/* TODO: 드래그를 통한 업로드 기능 추가 */}
+                <StyledGuideText>
+                  <strong>클릭하여 업로드</strong> 또는 파일을 드래그 하세요
+                </StyledGuideText>{' '}
+                <StyledFileTypeText>
+                  JPG,PNG 파일만 가능(최대 5MB)
+                </StyledFileTypeText>
+              </StyledContentWrapper>
+            </>
+          )}
+        </StyledProfileInputWrapper>
+      </StyledProfileWrapper>
+    </section>
+  );
+}
+
+export default ProfileSection;
+
+const StyledProfileWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  gap: 1.5rem;
+`;
+
+const StyledProfileInputWrapper = styled.label`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 1.8rem;
+
+  width: 100%;
+  height: fit-content;
+  padding: 4.3rem;
+  border: 3px dashed #e2e8f0;
+  border-radius: 16px;
+
+  background: #f8fafc;
```

Comment:
p3: 여기도 theme 을 사용하면 좋을 것 같아 👍

```suggestion
  border: 3px dashed ${({theme})=>theme.BG.LIGHT)}
```

Reply:
[style: theme 속성 사용하는 것으로 변경](https://github.com/woowacourse-teams/2025-Fit-toring/pull/496/commits/b214d9cfa1ffa653dec090f16dd7bd645e724cd2)